types/migrations/20230925072747_add_bookmarked_documents.sql

```diff
@@ -0,0 +1,3 @@
+-- Add migration script here
+alter table dailp_user
+  add column bookmarked_documents uuid[] not null default '{}';
```

Comment:
Like we mentioned in standup, we should use a join table here, something like what I wrote below:
```sql
create table user_bookmarked_document (
    document_id uuid not null references document (id) on delete cascade,
    user_id uuid not null references dailp_user (id) on delete cascade,
    bookmarked_on date not null DEFAULT now(),
    primary key (document_id, user_id) -- ensures that there is only one row per user/document pair 
);
```